### PR TITLE
feat: switch various components to use `@brightspace-ui/core`

### PR DESCRIPTION
**Pull request by devpow112**
All of these were simply drop in replacements as the files from the individual packages simply had a single line that was an import from `@brightspace-ui/core`.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @mtjandra
```
Comment (devpow112) [inline .github/CODEOWNERS:1]: This was just moved.

File: components/d2l-profile-image-base.js [hunk 2]
```diff
@@ -44,7 +43,7 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-profile-image-base">
 		</style>
 		<template is="dom-if" if="[[_showShadyPerson(_displayType)]]">
 			<div class="d2l-profile-image-container shady-person">
-				<d2l-icon icon="d2l-tier3:profile-pic"></d2l-icon>
+				<d2l-icon icon="tier3:profile-pic"></d2l-icon>
```
Comment (devpow112) [inline components/d2l-profile-image-base.js:46]: Removed `d2l-` from icon name as it's not needed.

File: package.json [hunk 3]
```diff
@@ -33,20 +33,11 @@
     "whatwg-fetch": "^2.0.0"
   },
   "version": "2.6.0",
-  "resolutions": {
```
Comment (devpow112) [inline package.json:36]: Not used

File: package.json [hunk 4]
```diff
@@ -33,20 +33,11 @@
...
-    "inherits": "2.0.3",
-    "samsam": "1.1.3",
-    "supports-color": "3.1.2",
-    "type-detect": "1.0.0"
-  },
-  "main": "d2l-users.js",
```
Comment (devpow112) [inline package.json:42]: Doesn't exist.

File: .github/CODEOWNERS (hunk 1 above)
Comment (dlockhart) [reply .github/CODEOWNERS:1]: Miles doesn't work here anymore, so probably better to just delete.

**Review by dlockhart — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (devpow112) [reply .github/CODEOWNERS:1]: K will update

**Review by dlockhart — APPROVED** [review]

**Pull request merged**